2020-07-29 13:42:24,987 INFO MainThread-140736581235648: Current user: True [in routes.py:141]
2020-07-29 13:42:24,993 INFO MainThread-140736581235648: Current user: False [in routes.py:143]
2020-07-29 13:42:25,003 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,018 INFO MainThread-140736581235648: Logged in user: patrick@gmail.com [in routes.py:117]
2020-07-29 13:42:25,019 INFO MainThread-140736581235648: Invalid next path in login request: http://www.badsite.com [in routes.py:126]
2020-07-29 13:42:25,035 INFO MainThread-140736581235648: Email address confirmed for: patrick@gmail.com [in routes.py:175]
2020-07-29 13:42:25,040 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,056 INFO MainThread-140736581235648: Confirmation link received for a confirmed user: patrick@gmail.com [in routes.py:168]
2020-07-29 13:42:25,061 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,071 INFO MainThread-140736581235648: Confirmation link received for a confirmed user: patrick@gmail.com [in routes.py:168]
2020-07-29 13:42:25,076 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,089 INFO MainThread-140736581235648: Invalid or expired confirmation link received from IP address: 127.0.0.1 [in routes.py:161]
2020-07-29 13:42:25,125 INFO MainThread-140736581235648: Logged in user: patrick@gmail.com [in routes.py:117]
2020-07-29 13:42:25,157 INFO MainThread-140736581235648: Duplicate login attempt by user: patrick@gmail.com [in routes.py:105]
2020-07-29 13:42:25,163 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,188 INFO MainThread-140736581235648: Logged out user: patrick@gmail.com [in routes.py:140]
2020-07-29 13:42:25,188 INFO MainThread-140736581235648: Current user: True [in routes.py:141]
2020-07-29 13:42:25,189 INFO MainThread-140736581235648: Current user: False [in routes.py:143]
2020-07-29 13:42:25,193 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,224 INFO MainThread-140736581235648: Logged in user: patrick@gmail.com [in routes.py:117]
2020-07-29 13:42:25,250 INFO MainThread-140736581235648: Duplicate login attempt by user: patrick@gmail.com [in routes.py:105]
2020-07-29 13:42:25,255 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,271 INFO MainThread-140736581235648: Logged out user: patrick@gmail.com [in routes.py:140]
2020-07-29 13:42:25,271 INFO MainThread-140736581235648: Current user: True [in routes.py:141]
2020-07-29 13:42:25,272 INFO MainThread-140736581235648: Current user: False [in routes.py:143]
2020-07-29 13:42:25,277 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,365 INFO MainThread-140736581235648: Logged in user: patrick@gmail.com [in routes.py:117]
2020-07-29 13:42:25,395 INFO MainThread-140736581235648: Logged out user: patrick@gmail.com [in routes.py:140]
2020-07-29 13:42:25,396 INFO MainThread-140736581235648: Current user: True [in routes.py:141]
2020-07-29 13:42:25,396 INFO MainThread-140736581235648: Current user: False [in routes.py:143]
2020-07-29 13:42:25,400 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,427 INFO MainThread-140736581235648: Logged in user: patrick@gmail.com [in routes.py:117]
2020-07-29 13:42:25,447 INFO MainThread-140736581235648: Password updated for user: patrick@gmail.com [in routes.py:248]
2020-07-29 13:42:25,482 INFO MainThread-140736581235648: Logged out user: patrick@gmail.com [in routes.py:140]
2020-07-29 13:42:25,483 INFO MainThread-140736581235648: Current user: True [in routes.py:141]
2020-07-29 13:42:25,483 INFO MainThread-140736581235648: Current user: False [in routes.py:143]
2020-07-29 13:42:25,488 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,501 INFO MainThread-140736581235648: Logged in user: patrick@gmail.com [in routes.py:117]
2020-07-29 13:42:25,560 INFO MainThread-140736581235648: Logged out user: patrick@gmail.com [in routes.py:140]
2020-07-29 13:42:25,560 INFO MainThread-140736581235648: Current user: True [in routes.py:141]
2020-07-29 13:42:25,561 INFO MainThread-140736581235648: Current user: False [in routes.py:143]
2020-07-29 13:42:25,573 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,615 INFO MainThread-140736581235648: Logged in user: patrick@gmail.com [in routes.py:117]
2020-07-29 13:42:25,636 INFO MainThread-140736581235648: Email re-sent to confirm email address for user: patrick@gmail.com [in routes.py:270]
2020-07-29 13:42:25,654 INFO MainThread-140736581235648: Logged out user: patrick@gmail.com [in routes.py:140]
2020-07-29 13:42:25,654 INFO MainThread-140736581235648: Current user: True [in routes.py:141]
2020-07-29 13:42:25,655 INFO MainThread-140736581235648: Current user: False [in routes.py:143]
2020-07-29 13:42:25,660 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,711 INFO MainThread-140736581235648: Starting the Flask Stock Portfolio App... [in __init__.py:98]
2020-07-29 13:42:25,712 INFO MainThread-140736581235648: Creating database tables in test_client fixture... [in conftest.py:158]
2020-07-29 13:42:25,724 INFO MainThread-140736581235648: Calling the index() function. [in routes.py:68]
2020-07-29 13:42:25,849 INFO MainThread-140736581235648: Starting the Flask Stock Portfolio App... [in __init__.py:98]
2020-07-29 13:42:25,872 INFO MainThread-140736581235648: Error! Received unexpected status code (404) when retrieving stock data (AAPL)! [in models.py:81]
2020-07-29 13:42:26,124 INFO MainThread-140736581235648: Error! Received unexpected status code (404) when retrieving weekly stock data (AAPL)! [in models.py:127]
